argus/action_manager/windows.py

```diff
@@ -527,12 +533,146 @@ def prepare_config(self, cbinit_conf, cbinit_unattend_conf):
         """
         super(WindowsNanoActionManager, self).prepare_config(
             cbinit_conf, cbinit_unattend_conf)
-        cbinit_conf.set_conf_value("stop_service_on_exit", False)
+        for conf in (cbinit_conf, cbinit_unattend_conf):
+            conf.conf.remove_option("DEFAULT", "logging_serial_port_settings")
+            conf.set_conf_value("stop_service_on_exit", False)
+
+    def get_installation_script(self):
+        """Get instalation script for CloudbaseInit."""
+        self.download_resource("windows/nano_server/installCBinit.ps1",
+                               self._INSTALL_SCRIPT)
+
+    def _unzip(self, zip_file, out_file, force=False):
+        """Unzip the file.
+
+        :param zip_file:
+            Path to zip file.
+        :param out_file:
+            Path to output file from unzipping process.
+        :param force:
+            If we should override the output file or not.
+        """
+
+        if self.exists(out_file) and not force:
+            raise exceptions.ArgusCLIError("Out file path already exists.")
+        elif self.exists(out_file):
+            (self.rmdir if self.exists(out_file) else self.remove)(out_file)
```

Comment:
Instead of `exists()` it should be `is_dir()`.
I would write this line as follows: `(self.remove if self.is_file(out_file) else self.rmdir)(out_file)`.